### DISCUSSION TITLE
Do not log hexadecimal values

### DISCUSSION THREAD

**Discussion by mike1819** [DirectoryTree/LdapRecord#488] [Q&A] [answered]
Hi all,
I configured the library to allow ldap users access to the application. Looking at the logs, I noticed that in the various stages of authentication (attepting, bound, ...) the LDAP values are displayed like this: `(cn=\4a\6f\68\6e\20\44\6f\65)`. How is possible to have the value **not** in hexadecimal but with readable value like `(cn=John Doe)`?  Can be very useful for debugging purposes
 
Thank!

**Comment by stevebauman** [comment, marked as answer]
Hi @mike1819, unescaped queries were originally included in logs, but this actually broke file logs and some other logging systems due to unescaped filters containing binary when querying by ActiveDirectory Object GUID's.

This was originally mentioned and resolved in this issue: https://github.com/DirectoryTree/LdapRecord/issues/293

**Reply by mike1819** [reply]
Thank you @stevebauman for the answer.

I changed hexadecimal with something human readable.
